_docs/00-getting-started.md

```diff
@@ -5,23 +5,29 @@ layout: docs
 permalink: /docs/getting-started.html
 ---
 
-## Get Infer
+## How to Download and Install Infer
 
 You can use Homebrew (Mac only), our binary releases, build infer from
-source, or use our Docker image.
+source, or use our Docker image. As of April 2019 (version 0.16.0),
+**it will require about 1.2 GB of disk space.**
 
-On Mac, the simplest way is to use [Homebrew](http://brew.sh/). Type this into a terminal:
+# Use Homebrew (macOS only)
```

Comment:
Should this be `###` instead of `#`?

## Get Infer
### Use Homebrew (macOS only)

Reply:
I don't know what the style guides are... I'll change it though.